ADA/databricks_fundamentals.qmd

```diff
@@ -1,14 +1,14 @@
 ---
-title: "Databricks fundamentals"
+title: "Databricks Fundamentals"
```

Comment:
Are we sure we want everything capitalised? Currently in other places we don't 
![image](https://github.com/user-attachments/assets/bc90f105-423b-42a4-b40f-6d17af82c873)

We're pretty inconsistent tbh, so should pick one way or the other?

Reply:
![image](https://github.com/user-attachments/assets/f91a2205-5ddb-426a-b034-400ca12e25b5)

I think I'd lean towards sentence case instead of title given how we use it elsewhere, though interested to know your thoughts?

Reply:
I think you are correct - it was all mixed up when I put the pages together and I chose to go in the wrong direction to fix it 😆